javascript/javascript1/week4/Homework.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  
+</body>
```

Comment:
Your HTML file should have only one `<body> ... </body>` tag and only one `</html>` tag